Keas.Mvc/ClientApp/components/People/PeopleContainer.tsx

```diff
@@ -18,33 +24,262 @@ export default class PeopleContainer extends React.Component<{}, IState> {
+    //     method: "POST"
+    //   });
+    // }
+
+    // const index = this.state.equipment.findIndex(x => x.id === equipment.id);
+    // if (index !== -1) {
+    //   // update already existing entry in equipment
+    //   const updateEquipment = [...this.state.equipment];
+    //   updateEquipment[index] = equipment;
+
+    //   this.setState({
+    //     ...this.state,
+    //     equipment: updateEquipment
+    //   });
+    // } else {
+    //   this.setState({
+    //     equipment: [...this.state.equipment, equipment]
+    //   });
+    // }
+  };
+
+  private _revokeEquipment = async (equipment: IPerson) => {
+    // // call API to actually revoke
+    // const removed: IEquipment = await this.context.fetch(`/api/${this.context.team.name}/equipment/revoke`, {
+    //   body: JSON.stringify(equipment),
+    //   method: "POST"
+    // });
+
+    // // remove from state
+    // const index = this.state.equipment.indexOf(equipment);
+    // if (index > -1) {
+    //   const shallowCopy = [...this.state.equipment];
+    //   if (this.props.person == null) {
+    //       // if we are looking at all equipment, just update assignment
+    //    shallowCopy[index].assignment = null;
+    //   } else {
+    //     // if we are looking at a person, remove from our list of equipment
+    //     shallowCopy.splice(index, 1);
+    //   }
+    //   this.setState({ equipment: shallowCopy });
+    // }
+  };
+
+  private _editEquipment = async (equipment: IPerson) =>
+  {
+    // const index = this.state.equipment.findIndex(x => x.id === equipment.id);
+
+    // if(index === -1 ) // should always already exist
+    // {
+    //   return;
+    // }
+
+    // const updated: IEquipment = await this.context.fetch(`/api/${this.context.team.name}/equipment/update`, {
+    //   body: JSON.stringify(equipment),
+    //   method: "POST"
+    // });
+
+    // updated.assignment = equipment.assignment;
+
+    // // update already existing entry in key
+    // const updateEquipment = [...this.state.equipment];
+    // updateEquipment[index] = updated;
+
+    // this.setState({
+    //   ...this.state,
+    //   equipment: updateEquipment
+    // }); 
+  }
+
+  // managing counts
+  private _assetAssigned = (type: string, spaceId: number, personId: number, created: boolean, assigned: boolean) => {
+    const index = this.state.people.findIndex(x => x.id === personId);
+    if(index > -1)
+    {
+        const people = [...this.state.people];
+        switch(type) {
+          case "equipment": 
+            people[index].equipmentCount++;
+            break;
+          case "key":
+            people[index].keyCount++;
+            break;
+          case "access":
+            people[index].accessCount++;
+            break;
+          case "workstation":
+            people[index].workstationCount++
+        }
+        this.setState({people});
+    } 
+}
+
+private _assetRevoked = (type: string, spaceId: number, personId: number) => {
```

Comment:
@laholstege probably you can combine these too methods to keep it DRY.  Maybe _assetCountUpdated() where you pass along the personId and the delta and then just do += math